lib/server.js

```diff
@@ -15,7 +15,9 @@
 
 'use strict';
 
-var _ = require('lodash');
+var isString = require('lodash/isString');
```

Comment:
@zetlen I agree that you save some bytes and speed things up by a faction of a millisecond, but don't you feel it's a premature optimization?

Minimizing bundle size on the client is an excellent argument, but is this file even getting bundled at all? In the [example](https://github.com/paypal/react-engine/blob/master/example/public/index.js#L22), it specifically requires `./lib/client`.